js/nativeModule.js

```diff
@@ -0,0 +1,17 @@
+import { NativeModules } from 'react-native';
+const RNKInAppUpdate = NativeModules.RNKInAppUpdate;
+
+if (Platform.OS === 'ios') {
+  console.warn(`@kyivstarteam/react-native-in-app-update: Module does not support ios platform`);
+}
+
+if (!RNKInAppUpdate) {
+  throw new Error(
+    `@kyivstarteam/react-native-in-app-update: NativeModule.RNKInAppUpdate is null. To fix this issue try these steps:
+    • For react-native <= 0.59: Run \`react-native @kyivstarteam/react-native-in-app-update\` in the project root.
```

Comment:
react-native @kyivstarteam/react-native-in-app-update => react-native link @kyivstarteam/react-native-in-app-update
Replace pls

Reply:
done